web/models/email.ex

```diff
@@ -0,0 +1,50 @@
+defmodule Askbywho.Email do
+  use Askbywho.Web, :model
+  alias Askbywho.Repo
+  alias Askbywho.Brand
+
+  schema "emails" do
```

Comment:
Added a trello card on this https://trello.com/c/oV3auVsy/35-plan-user-management